src/main/java/com/ldtteam/structurize/Structurize.java

```diff
@@ -42,6 +44,8 @@ public Structurize()
         Mod.EventBusSubscriber.Bus.MOD.bus().get().register(LifecycleSubscriber.class);
         Mod.EventBusSubscriber.Bus.FORGE.bus().get().register(EventSubscriber.class);
         DistExecutor.runWhenOn(Dist.CLIENT, () -> () -> Mod.EventBusSubscriber.Bus.FORGE.bus().get().register(ClientEventSubscriber.class));
+
+        BlockSolidSubstitution.NOT_SOLID.add(blockState -> !blockState.getMaterial().isSolid() || blockState.getBlock() instanceof AirBlock);
```

Comment:
Probably move this to proper loading phase, rather than mod constructor

Reply:
To which one would you move this?

Reply:
I don't know :D probably block registration since registries are before common setup, or even static block code of ModBlocks class